Replicator/tests/ReplicatorAPITest.cc

```diff
@@ -618,7 +614,6 @@ TEST_CASE_METHOD(ReplicatorAPITest, "Pending Document IDs Non-Existent Collectio
 
     c4repl_start(_repl, false);
     REQUIRE_BEFORE(5s, c4repl_getStatus(_repl).level == kC4Stopped);
-    encodedDocIDs = c4repl_getPendingDocIDs(_repl, Republic, &err);
```

Comment:
How is this still going to pass?  The next line is checking the result of this error (perhaps it should also be checking the result of `encodedDocIDs`?)